styles/index.scss

```diff
@@ -32,20 +48,84 @@ body {
   z-index: -1;
 }
 
+section {
+  margin: 0 auto;
+  padding: 82px 24px 0;
+
+  @container body (min-width: 768px) {
+    padding: 200px 72px 0;
+    max-width: 1040px;
+  }
+
+  @container body (min-width: 1440px) {
+    padding: 200px 200px 0;
+  }
+
+  div > div {
+    @container body (min-width: 1024px) {
+      display: flex;
+      justify-content: space-between;
+    }
+  }
+}
+
 h1 {
   font-size: 72px;
   font-weight: 400;
+  letter-spacing: 0;
+  line-height: 1.1944; // 86px
 
   @container body (min-width: 1024px) {
     font-size: 96px;
   }
 }
 
-p {
-  font-size: 16px;
-  font-weight: 300;
-  line-height: 32px;
-  letter-spacing: 0.05em;
+h2 {
+  font-size: 3rem; // 48px
+  font-weight: 400;
+  line-height: 120.8333%; // 58px
+  margin: 0 0 24px -10px;
+
+  @container body (min-width: 1024px) {
+    font-size: 3.75rem; // 60px
+    line-height: 120%; // 72px
+  }
+
+  .mark-teal {
+    font-weight: 400;
+    display: block;
+    padding: 0 0.5908rem 2.125rem 0.5908rem;
+    width: fit-content;
+  }
+
+  span {
+    margin: 0 0 30px 10px;
+    display: block;
+  }
+}
+
+h3 {
+  display: flex;
+  align-items: center;
+  font-weight: 500;
+  font-size: 2rem; // 32px
+  line-height: 150%; // 48px
+  padding-top: 64px;
+
+  @container body (min-width: 1024px) {
+    padding-top: 202px;
+  }
```

Comment:
I think these styles may be too specific for the global `h3` styles? I noticed for "debriefing" and "briefing" you added `padding: 0` to override these, it might be easier to maintain if we created a separate class or someting for `h3`s that need top padding?

Reply:
I think we always need the 64px top padding for the h3s, but good call on the larger screen sizes being different--I've moved those out to separate classes.